packages/react-components/react-positioning/src/createSlideStyles.ts

```diff
@@ -0,0 +1,84 @@
+import { tokens } from '@fluentui/react-theme';
```

Comment:
Lives in the same package as the arrow styles
https://github.com/microsoft/fluentui/blob/8b699accbe90425a8f3042b0943ac20d457d9bcc/packages/react-components/react-positioning/src/createArrowStyles.ts#L63